server/graphql-schema.js

```diff
@@ -685,6 +685,10 @@ const graphqlSchema = gql`
         found on the TestPlanVersion type.
         """
         testResults: [TestResult]!
+        """
+        The number of completed tests for this TestPlanRun.
+        """
+        testResultsLength: Int!
```

Comment:
Maybe explain why this is needed in the docs, i.e. the tests take a long time to be returned

Reply:
Sure, can do